Jenkinsfile

```diff
@@ -13,9 +15,15 @@ To configure the repo/branch go to:
                    for nonexistent branches.
         * Project Repository: Url to the shared lib
         * Credentials: SSH key to access the repo
-
-Updating the shared repo will take affect on the next pipeline invocation.
-The "_" denotes that all modules will be imported from the shared library.
 */ 
-@Library("vivarium_build_utils") _
-reusable_pipeline(scheduled_branches: ["main"], upstream_repos: ["layered_config_tree"])
+
+// Load the get_vbu_version function from vivarium_build_utils/boostrap/
+@Library("get_vbu_version@main") _
```

Comment:
this is the reason this build is failing...we need to get this into vivarium_build_utils main branch!